### PR TITLE
ANPR 

### PR DESCRIPTION
**Pull request by atharva2702**

### CODE REVIEW

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@atharva2702 @kush1920 Changes Required
1. Remove all tmp.md file
2. CAD Images have to be present inside Mechanical Design folder. Upload other project related pictures in Images folder.
3. Rename "anpr.jpg" as "Poster.jpg"
4. Justify text in Report.
5. workflow.jpg has to be present inside Images and Videos/Images.
6. In README.md, Cost Structure should be a H2 type heading.
7. In README.md, hyperlink Team Members and Mentors names with their GitHubprofile link.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,82 @@
...
+
+Shopping Malls : To keep a record of vehicles entering
+
+Toll Plaza : On highways for security reasons 
+
+Parking Plazas : To detect and generate tax for parked vehicles automatically
+
+Educational or Government Institutions : To ensure only authorized vehicles are permitted inside 
+
+## Limitations
```
Comment (agshubhs) [inline README.md]: @atharva2702 
This should be lists

File: README.md [hunk 2]
```diff
@@ -1,2 +1,82 @@
...
+
+Restrictions on Camera's Field Of View , Resolution an Frame Rate .
+A better OCR trained on strong GPU's will give better results .
+
+## Future Improvement
```
Comment (agshubhs) [inline README.md]: This should be  in list.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,82 @@
...
+### Cost Structure
+
+| Components       | Cost(INR.)     |
+| ---------------- |:--------------:|
+| Raspberry Pi     | 3830.00        |
+| Battery          | 1500.00        |
+| LOGITECH WEBCAM  | 3500.00        | 
+| Total            | 8830.00        |
+
+## Applications
```
Comment (agshubhs) [inline README.md]: use Unordered list for them also.

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@atharva2702 Changes Required
1. Alignment is not proper in the report. Fix it
2. In README.md, use `<p align="justify">Content<\p>` to justify the text, where content is the text you want to justify.

**Review by SanjeevKrishnan — APPROVED** [review]
@atharva2702 @kush1920 All requirements are met!
@Shubhanshu07 I suggest for merge!

**Comment by agshubhs** [comment]
@atharva2702 Great Work GUYS..!!
Merging it

**Pull request merged**